fuzz/cmp.c

```diff
@@ -155,6 +155,19 @@ static int process_pollReq(OSSL_CMP_SRV_CTX *srv_ctx,
     return 0;
 }
 
+static int reset_transaction(OSSL_CMP_SRV_CTX *srv_ctx)
+{
+    ERR_raise(ERR_LIB_CMP, CMP_R_ERROR_PROCESSING_MESSAGE);
```

Comment:
What is the point in throwing an error here?

Reply:
error message is removed.